internal/config_test.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFillAndValidate(t *testing.T) {
+	t.Parallel()
+
+	for name, tc := range map[string]struct {
+		in          *Results
+		expected    *Results
+		expectedErr error
+	}{
+		"FillAndValidate_Filled": {
```

Comment:
Would you please don't use the underscores in test names? I mean, for the code consistency.

Reply:
https://github.com/FerretDB/FerretDB/issues/694